Makefile

```diff
@@ -123,3 +123,13 @@ clone-samples:
 		tar -xJvf samples/linux/clean/$$file -C samples/linux/clean; \
 	done
 	tar -xJvf samples/macOS/clean/bincapz.xz -C samples/macOS/clean
+
+ARCH ?= $(shell uname -m)
+CRANE_VERSION=v0.20.2
+install-crane-$(ARCH)-$(CRANE_VERSION):
+	mkdir -p out
+	GOBIN=$(CURDIR)/out go install github.com/google/go-containerregistry/cmd/crane@$(CRANE_VERSION)
+
+.PHONY: extract-image
+extract-image: install-crane-$(ARCH)-$(CRANE_VERSION)
```

Comment:
"extract" doesn't feel like the right word for this action - maybe "dump" or "export"?

Reply:
Updated in [`f59bdef` (#438)](https://github.com/chainguard-dev/bincapz/pull/438/commits/f59bdef7cce505502d1550ee5295604d457b63e1).